attractions/utils/classes.js

```diff
@@ -1,9 +1,19 @@
 const consecutiveSpace = / {2,}/;
 
+/**
+ * Removes extra whitespace in class strings
+ * @param {string} classString The string to be trimmed
+ * @returns {string} Clean class string
+ */
 export function stripClassWhitespace(classString) {
   return classString.replace(consecutiveSpace, ' ').trimRight();
 }
 
+/**
+ * Filters out falsy classes
+ * @param  {...string|false} args The classes to be filtered
```

Comment:
```suggestion
 * @param {...Array<string | false>} args The classes to be filtered
```
Shouldn't it be like that?

Reply:
No, the `...` already implies it's an array